tests/Oryx.Integration.Tests/DotNetCoreEndToEndTests.cs

```diff
@@ -487,6 +487,8 @@ public static TheoryData<string> StartupCommandData
 
                 data.Add($"'dotnet {tempAppDir}/{NetCoreApp21WebApp}.dll'");
                 data.Add($"'echo \"foo bar\" && dotnet {tempAppDir}/{NetCoreApp21WebApp}.dll'");
+                data.Add($"'bash -c \"echo foo && dotnet {tempAppDir}/{NetCoreApp21WebApp}.dll\"'");
+                data.Add($"'key=\"a;b;c\" dotnet {tempAppDir}/{NetCoreApp21WebApp}.dll'");
```

Comment:
@gpcastro Based on the file you shared, I didn't see much variety in them to be concerned about. I did not see usage of single quotes at all in that data, so I feel we are covered for all the cases mentioned in that document.